src/saturn/imgui/saturn_imgui_machinima.cpp

```diff
@@ -411,7 +410,7 @@ void imgui_machinima_quick_options() {
     ImGui::Checkbox("Fog", &enable_fog);
     imgui_bundled_tooltip("Toggles the fog, useful for near-clipping shots");
     ImGui::Checkbox("Rainbow DisplayLists", &rainbow);
-    imgui_bundled_tooltip("Sets the displaylists to be Rainbow.");
+    imgui_bundled_tooltip("Makes the displaylists Rainbow.");
```

Comment:
can you make the "rainbow" not capitalized plz

Reply:
> can you make the "rainbow" not capitalized plz

no

Reply:
![death](https://cdn.discordapp.com/emojis/1246885973455933583.webp?size=48&quality=lossless&name=death)

Reply:
why am i here